utils/launch.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
Why restrict ourselves to just Posix shell? Requiring bash is hardly a massive burden.